frontend/src/InteractiveImport/Interactive/InteractiveImportModalContent.js

```diff
@@ -53,6 +53,16 @@ const columns = [
     label: () => translate('Tracks'),
     isVisible: true
   },
+  {
+    name: 'isSingleFileRelease',
```

Comment:
I don't think we need two new columns here, either add isSingleFileRelease with mouseover that shows the cue path, or just add the cue path. Having both as columns doesn't add much value in my mind.

Reply:
I'll only keep the cue path there.